packages/airnode-admin/test/e2e/cli.feature.ts

```diff
@@ -660,4 +661,19 @@ describe('CLI', () => {
       expect(out).toEqual(`Airnode address: ${airnodeWallet.address}`);
     });
   });
+
+  describe('has valid examples', () => {
+    const exampleOutcomes = [
+      'Airnode address: 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266',
+      'Sponsor wallet address: 0x61cF9Eb3691A715e7B2697a36e9e60FdA40A8617',
+      'Endpoint ID: 0x901843fb332b24a9a71a2234f2a7c82214b7b70e99ab412e7d1827b743f63f61',
+    ];
+
+    test.each(cliExamples)(`example command: %# is valid`, (example: string) => {
```

Comment:
Won't `%#` result in an extremely long test message? I'd just use index instead.

Reply:
I actually dislike a bit the `test.each` helper and would prefer this instead:

```
cliExamples.forEach((command, index) => {
it('tests example command ${index}', () => {...})
})
```

Reply:
Oh, I thought it's interpolating the command. (this is also a reason I don't like it - I find it hard to remember what is interpolated...)